src/main/java/temporal/ChildWorkflowImpl.java

```diff
@@ -20,9 +23,13 @@ public void start() {
 
   @Override
   public void sendNotification(OrderInfo orderInfo) {
-    logger.info(
-        "Sending notification for customer from child workflow: " + orderInfo.getCustomerId());
-    Workflow.sleep(Duration.ofSeconds(3));
-    exit = true;
+    try {
+      String json = objectMapper.writeValueAsString(orderInfo);
```

Comment:
The OrderInfo class should implement a `toString()` method.  There is no need to use `objectMapper`, rather just call `orderInfo.toString()`.

Reply:
In the meantime until I hear back from you, @pvsone I'll override` .toString()` in the `OrderInfo` class.

Reply:
[Here is a link](https://github.com/reselbob/SimpleTemporalActorModel/blob/25f45863c51180bd32dad06a60b9056d046a7c26/src/main/java/temporal/model/OrderInfo.java#L48) to the simplified implementation of .toString() override. If there is a better way to do this, please don't hesitate to let me know, @pvsone

Reply:
When I tried to use ObjectMapper I got errors seemingly from the internal Jackson serializer. Maybe I did something wrong, not sure. @pvsone

Reply:
The right approach is the one you took - to override .toString() in the OrderInfo class.